keps/sig-cli/kustomize-reorder-plugins.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Kustomize Resource Plugins Based Reodering
+authors:
+ - "@jbrette"
+owning-sig: sig-cli
+participating-sigs:
+ - sig-apps
+ - sig-api-machinery
+reviewers:
+ - "@monopole"
+ - "@Liujingfang1"
+approvers:
+ - "@monopole"
+editors:
+ - "@jbrette"
+creation-date: 2019-09-01
+last-updated: 2019-09-01
+status: proof of concept
```

Comment:
Change the status to `provisional`. You can take a look at the KEP template [here](https://github.com/kubernetes/enhancements/blob/master/keps/YYYYMMDD-kep-template.md).

Reply:
done